helpers/pagination.js

```diff
@@ -0,0 +1,16 @@
+const pagination = (page, size, count) => {
+  let currentPage = Math.abs(Number(page)) || 1;
+  let offset = 0;
+  let limit = 2;
```

Comment:
hello @pitaz nice work so far, but from the API spec, the limit should be 20 articles.

Reply:
Thanks @azupatrick0 This has been resolved.

Reply:
Kudos @pitaz